common/include/limits.h

```diff
@@ -72,11 +72,11 @@
 /* Minimum and maximum values a `signed long int' can hold.
    (Same as `int').  */
 #ifndef __LONG_MAX__
-#ifdef __R5900
+#if defined(__R5900) && (__GNUC__ <= 3)
```

Comment:
Are you aware of a similar problem in newlib? See this commit:
https://gitlab.com/ps2max/toolchain/newlib/-/commit/3c4ebfcea1fb06f0d5600b134198506ee2c274af

Reply:
My patch of newlib doesn't change `__LONG_MAX__`.